### DISCUSSION TITLE
Infinite loading for suggestions

### DISCUSSION THREAD

**Discussion by emilgerz** [microsoft/vscode-discussions#699] [Extension Development QnA] [answered]
Hey, I have some problem here with suggestions. I usually open my VS code app and its all good, but after may be 30 seconds my suggestions is going to infinite loading like in the screen (translate: there is wrote 'Loading...' in russian)

[Screen](https://sun9-67.userapi.com/impg/7ToJ6NvobDLPclRx8_1YINDEO9Bvbqkyt_QBZA/Bb2IjAvnEBQ.jpg?size=818x160&quality=96&sign=487eb99e7787d505787b40783c5e0983&type=album)

How can i fix that?

**Comment by burkeholland** [comment, marked as answer]
This things are likely caused by an extension. Run "Extension Bisect" from the command palette and you can find the culprit.
